src/lib/combat_achievements/easy.ts

```diff
@@ -38,7 +38,7 @@ export const easyCombatAchievements: CombatAchievement[] = [
 		monster: 'Barrows',
 		desc: 'Kill any Barrows Brother using only magical damage.',
 		rng: {
-			chancePerKill: 4,
+			chancePerKill: 1,
 			hasChance: isCertainMonsterTrip(Monsters.Barrows.id)
```

Comment:
maybe enforce training magic for consistence with other tasks of this nature?

See: https://github.com/oldschoolgg/oldschoolbot/blob/8167f9ffcd5c73800d7cb272bec73a81a6e7e3e5/src/lib/combat_achievements/medium.ts#L134C5-L134C54